osu.Game/Audio/PreviewTrackManager.cs

```diff
@@ -90,7 +88,7 @@ public void StopAnyPlaying(IPreviewTrackOwner source)
         public class TrackManagerPreviewTrack : PreviewTrack
         {
             [Resolved(canBeNull: true)]
-            public IPreviewTrackOwner Owner { get; private set; }
+            public IPreviewTrackOwner? Owner { get; private set; }
```

Comment:
`canBeNull: true` can be removed from the attribute since https://github.com/ppy/osu-framework/pull/5224 (the NRT spec on the type suffices).